readme.md

```diff
@@ -34,10 +34,59 @@ Alternatively the project can be run in a container through the docker-compose.y
 # Kubernetes
 The service has been developed with the intention of running in Kubernetes.  A helm chart is included in the `.\helm` folder.
 
+# Basic Authentication
+The ingress controller may be protected with basic authentication by setting the `auth` value exposed by [values.yaml](./helm/values.yaml) on deployment.
+
+The provided auth string must be an htpasswd encoded for use in the data field of a Kubernetes secret.
+
+To generate the correct format auth token first create a username and password using htpasswd.
+Below shows creating a password for the user 'defra'.
+
+`htpasswd -c ./auth defra`

```

Comment:
Should we add "auth" to the .gitignore list?

Reply:
I've added auth to both the .gitignore and the .dockerignore